src/search.jl

```diff
@@ -62,30 +69,58 @@ function find(
     map(
         x -> ICESat_Granule(
             Symbol(product),
-            x["producer_granule_id"],
-            get(get(x, "links", [Dict()])[1], "href", ""),
-            icesat_info(x["producer_granule_id"]),
+            x["filename"],
+            x["https"],
+            x["s3"],
+            (min_x = [], min_y = [], max_x = [], max_y = []),
+            icesat_info(x["filename"]),
         ),
         granules)
 end
 
 function parse_cmr_json(r)
     data = JSON.parse(String(r.body))
-    get(get(data, "feed", Dict()), "entry", [])
+
+    # each granule is a column in the vector
+    f = [granule_info(row[1]) for row in eachrow(get(data, "items", Dict()))]
```

Comment:
As this is UMM, I believe `items` is always present, so this is more straightforward and enables us to the JSON dependency and only rely on JSON3.
```suggestion
    data = JSON3.read(r.body)
    map(granule_info, data.items)
```
Also note the `.` notation instead of `get()`, which could also apply to subsequent filtering.